### PR TITLE
ED-1227: Add oci oss oss support for kp/flinkjobs -- superseed #695 , #713

### PR DESCRIPTION
**Pull request by ddevadat**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by ddevadat** [comment]
@pallakartheekreddy this is the PR for kp jobs

File: video-stream-generator/src/main/scala/org/sunbird/job/videostream/helpers/OCIRequestBody.scala [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package org.sunbird.job.videostream.helpers
+
+object OCIRequestBody {
+  val get_object_parameters = "{\"taskParameters\": [{\"storageType\": \"objectStorage\",\"target\": \"getFiles/${/input/objectName}\",\"namespaceName\": \"${/input/namespaceName}\",\"bucketName\": \"${/input/bucketName}\",\"objectName\": \"${/input/objectName}\"}]}"
```
Comment (maheshkumargangula) [inline video-stream-generator/src/main/scala/org/sunbird/job/videostream/helpers/OCIRequestBody.scala:4]: Please consider using the Scala string format """{}""" to have a readable format of the JSON data.

File: credential-generator/collection-cert-pre-processor/src/main/scala/org/sunbird/job/collectioncert/functions/IssueCertificateHelper.scala [hunk 2]
```diff
@@ -27,9 +27,10 @@ trait IssueCertificateHelper {
         val enrolledUser: EnrolledUser = validateEnrolmentCriteria(event, criteria.getOrElse(config.enrollment, Map[String, AnyRef]()).asInstanceOf[Map[String, AnyRef]], certName, additionalProps)(metrics, cassandraUtil, config)
         //validateAssessmentCriteria
         val assessedUser = validateAssessmentCriteria(event, criteria.getOrElse(config.assessment, Map[String, AnyRef]()).asInstanceOf[Map[String, AnyRef]], enrolledUser.userId, additionalProps)(metrics, cassandraUtil, contentCache, config)
+        logger.info("assessedUser =>"+assessedUser)
         //validateUserCriteria
         val userDetails = validateUser(assessedUser.userId, criteria.getOrElse(config.user, Map[String, AnyRef]()).asInstanceOf[Map[String, AnyRef]], additionalProps)(metrics, config, httpUtil)
-
+        logger.info("userDetails =>"+userDetails)
```
Comment (maheshkumargangula) [inline credential-generator/collection-cert-pre-processor/src/main/scala/org/sunbird/job/collectioncert/functions/IssueCertificateHelper.scala:33]: Remove logging of user details.

Comment (ddevadat) [reply credential-generator/collection-cert-pre-processor/src/main/scala/org/sunbird/job/collectioncert/functions/IssueCertificateHelper.scala:33]: removed logger info

File: video-stream-generator/src/main/scala/org/sunbird/job/videostream/helpers/OCIRequestBody.scala (hunk 1 above)
Comment (ddevadat) [reply video-stream-generator/src/main/scala/org/sunbird/job/videostream/helpers/OCIRequestBody.scala:4]: Similar style is currently followed for AWS, Azure request body. Can we take this up in the future versions

Comment (maheshkumargangula) [reply video-stream-generator/src/main/scala/org/sunbird/job/videostream/helpers/OCIRequestBody.scala:4]: Okay.

**Comment by ddevadat** [comment]
> 

@pallakartheekreddy i tried the suggestion, but the circle ci failed at Installation of imagemagick

**Review by maheshkumargangula — APPROVED** [review]

**Pull request merged**